sources/api/netdog/test_data/networkd/network/eno3.network

```diff
@@ -0,0 +1,13 @@
+[Match]
+Name=eno3
+[Link]
+RequiredForOnline=true
+RequiredFamilyForOnline=ipv4
+[Network]
+DHCP=ipv4
+[DHCPv4]
+UseDNS=true
+UseDomains=true
+[DHCPv6]
+UseDNS=true
+UseDomains=true
```

Comment:
I would expect the configuration to disable this if `dhcp6 = false` in our config? Or are these innocuous to networkd under the hood?

Reply:
Good point - I overlooked that.  I'll leave those entries out if DHCP6=false